pulsar.sh

```diff
@@ -152,26 +126,24 @@ if [ $OS == 'Mac' ]; then
 elif [ $OS == 'Linux' ]; then
   SCRIPT=$(readlink -f "$0")
 
-  case $CHANNEL in
-    beta)
-      PULSAR_PATH="/opt/Pulsar-beta/pulsar"
-      ;;
-    nightly)
-      PULSAR_PATH="/opt/Pulsar-nightly/pulsar"
-      ;;
-    dev)
-      PULSAR_PATH="/opt/Pulsar-dev/pulsar"
-      ;;
-    *)
-      PULSAR_PATH="/opt/Pulsar/pulsar"
-      ;;
-  esac
+  PULSAR_PATH="/opt/Pulsar/pulsar"
 
   #Will allow user to get context menu on cinnamon desktop enviroment
-  if [[ "$(expr substr $(printenv | grep "DESKTOP_SESSION=") 17 8)" == "cinnamon" ]]; then
-    cp "resources/linux/desktopenviroment/cinnamon/pulsar.nemo_action" "/usr/share/nemo/actions/pulsar.nemo_action"
+  #Add a check to make sure that DESKTOP_SESSION is set before attempting to grep it
+  #expr substr is expecting 3 arguments string, index, length
+  #If grep doesnt find anything is provides an empty string which causes the expr: syntax error: missing argument after '8' error
```

Comment:
```suggestion
  #If grep doesnt find anything is provides an empty string which causes the expr: syntax error: missing argument after '8' error - see pulsar-edit/pulsar#174
```
I know this is an unnecessary change, but something I feel helps a lot when reviewing code a year from now to help know why some decisions were made. Especially like you mention this was already done in a strange way, and we are adding more to that. So this can hopefully prevent our future selfs some confusion.